backend/src/apiserver/config/config.json

```diff
@@ -11,6 +11,10 @@
     "BucketName": "mlpipeline",
     "PipelinePath": "pipelines"
   },
+  "ArchiveConfig": {
+    "LogFileName": "step-main.log",
```

Comment:
Thanks @rafalbigaj, it looks great. Our current logs are pushed as `step-main.tgz`. If you think `.log` is better, we can update our default archive logs function to push it without compression.

Reply:
@Tomcli I have updated the default. Thanks!